portfolio.html

```diff
@@ -7,5 +7,60 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <div class="wrapper">
+      <header>
+        <div class="logo">
+          <a href="#">My Logo</a>
+        </div>
+        <nav>
+          <div class="menu-icon">
+            <img src="images/Union.png" alt="menu button" />
+          </div>
+        </nav>
+      </header>
+      <main>
+        <section class="headline">
+          <h2 class="title">I’m Yaseen Glad to see you!</h2>
+          <p class="details">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <p></p>
+          <h2 class="contact-title">LET’S CONNECT</h2>
```

Comment:
- [ ] [OPTIONAL] It's better if you convert this header into a link so you can link to external websites or add a link to send an email